app/models/user.rb

```diff
@@ -450,6 +451,10 @@ def hotness_score
     search_score
   end
 
+  def rate_limiter
```

Comment:
Made this public bc I think we could refactor other parts of the code like in the application controller `current_user.rate_limiter` rather than `RateLimitChecker.new(current_user)`